src/app/hearing/components/view-hearing/view-hearing.component.html

```diff
@@ -30,16 +30,21 @@
     </div>
 
     <div class="tile">
-      <strong>Estimated duration</strong><br>
-      {{formatDuration(hearing.duration)}}
+      <strong>Estimated duration
+      </strong><br>
+      {{formatDuration(hearing.duration)}}<ng-container *ngIf="hearing.isMultiSession; then days; else minutes"></ng-container>
```

Comment:
I think that you can use appDurationFormat pipe instead of using these method here. So probably you can remove the method from the component and just stick to pipe.

Reply:
I actually tried to use that pipe - but it'd mean i'd need to move it into a separate shared module to use it (see screenshot attached).
Would it be better if i created this module or shall i leave it as it is?
![screen shot 2018-11-07 at 15 00 58](https://user-images.githubusercontent.com/20964997/48135703-f69c9580-e29d-11e8-91f6-66ba10cc81d7.png)

Reply:
It should work with having Pipe declared just in CoreModule, will have a look in the morning into why there is a problem

Reply:
It's because Hearing is not a descendant of CoreModule 
<img width="663" alt="screen shot 2018-11-08 at 08 46 54" src="https://user-images.githubusercontent.com/20964997/48184642-f00c2d80-e332-11e8-8314-2edce05b0258.png">

Reply:
Try to import CoreModule into HearingModule.

Reply:
Done